Traceback (most recent call last):
  File "script.py", line 11, in <module>
    train_dataset = datasets.MNIST(
  File "/home/kshitij.cse22.itbhu/venv/lib/python3.8/site-packages/torchvision/datasets/mnist.py", line 102, in __init__
    raise RuntimeError("Dataset not found. You can use download=True to download it")
RuntimeError: Dataset not found. You can use download=True to download it
